benchmarking/config.yaml

```diff
@@ -0,0 +1,5 @@
+prod_mode: True # available values: True or False
```

Comment:
This should be set to False as default as we do with the other kits.

Reply:
good catch! I've already pushed this change, thanks!